User Doc for LPWAN Custom Codec feature (#206)
SoftwareAG/c8y-docs: content/protocol-integration/lpwan-custom-codec-bundle/overview.md

---
weight: 10
title: Overview
layout: redirect
---

{{< product-c8y-iot >}} has the ability to integrate LPWAN devices via LPWAN agents.
For a list of all supported LPWAN agents see the [Protocol integration guide](../overview/).

> The section explains how Cumulocity supports LPWAN Custom Codec microservice for decoding and encoding the device payload.

Reviewer: LPWAN custom codec microservice > the LPWAN custom codec microservice

---
weight: 10
title: Overview
layout: redirect
---

LPWAN Agents provide a way to plugin custom codecs, capable of encoding and decoding the device payloads to and from the LPWAN devices. 

> A custom codec is a typical Cumulocity microservice conforming to a predefined contract.

Reviewer 2: ```suggestion
A custom codec is a typical {{< product-c8y-iot >}} microservice conforming to a predefined contract.
```

> * [Abstract](#abstract) Why Cumulocity IoT platform supports LPWAN Codecs.

Reviewer 2: ```suggestion
* [Abstract](#abstract) Why {{< product-c8y-iot >}} platform supports LPWAN Codecs.
```

layout: redirect
---

{{< product-c8y-iot >}} has the ability to integrate LPWAN devices via LPWAN agents, for a list of all supported LPWAN agents see the [Protocol integration guide](https://cumulocity.com/guides/protocol-integration/overview/).

> The section explains how Cumulocity supports LPWAN custom codec microservice for decoding and encoding the device payload.

Reviewer 3: ```suggestion
The section explains how Cumulocity IoT supports LPWAN custom codec microservice for decoding and encoding the device payload.
```

Reviewer: @MWindel We should use a shortcode instead.

Reviewer 4: @BeateRixen @MWindel @rssag  Please hold on for a while until we submit the final version of the changes as we are rearranging the sections and some of the content.

Reviewer 2: @miryalarahul oh sorry, bad timing. i just committed a few changes :/ hope it won't cause too much trouble
